factories/creatorships.rb

```diff
@@ -0,0 +1,16 @@
+require "faker"
+FactoryBot.define do
+  factory :creatorships do
+    title { "My title is long enough" }
```

Comment:
What is this being used for?

Reply:
Looks like it was left over from a previous attempt at doing the tests with cucumber